### ISSUE TITLE
Save to Existing Coursepack (whole text)

### ISSUE THREAD

**Issue by tonyahowe** [LiteratureInContext/LiC-app#147]
Save whole text from search (search: "robinson crusoe") to existing coursepack by check box not working

https://user-images.githubusercontent.com/1153464/194079685-44862a1e-9f9c-492f-898b-1ff79a6a6421.mov

**Comment by wsalesky** [comment]
Can you try now?

**Comment by tonyahowe** [comment]
Nope--still can't do this (or selections) as thowe. Can as admin, though.

**Comment by wsalesky** [comment]
@tonyahowe This can be closed now, correct?
